
---
social_robot_arm_sdk/JointPose pose
